packages/react/src/utilities/ThemeProvider/makeStyles.ts

```diff
@@ -44,6 +44,8 @@ export type UseStylesOptions = {
  *
  * @param styleOrFunction - Either a css javascript object, or a function which takes in `ITheme`
  * and returns a css javascript object.
+ *
+ * @deprecated Use `mergeStyles` instead.
```

Comment:
It might be good to add a note that this only applies with v8, and that makeStyles is what we intend to use going forward but not this implementation

Reply:
Possibly also deprecate any related types? (but that's less important)

Reply:
I've noted that this only applies to v8 and I've deprecated related types.